test/e2e/network/kube_proxy.go

```diff
@@ -77,6 +77,16 @@ var _ = SIGDescribe("Network", func() {
 
 		zero := int64(0)
 
+		// Some distributions (Ubuntu 16.04 etc.) don't support the proc file.
+		_, err := framework.IssueSSHCommandWithResult(
+			"sudo ls /proc/net/nf_conntrack",
+			framework.TestContext.Provider,
+			clientNodeInfo.node)
+		if err != nil && strings.Contains(err.Error(), "No such file or directory") {
+			framework.Skipf("The node %s does not support /proc/net/nf_conntrack", clientNodeInfo.name)
```

Comment:
Is there no other way to accomplish what this test is trying to do?

Reply:
necro-ing a bit but ...
I'm pretty sure was not the correct way to do this, I definitely have conntrack and a working kind cluster on my machine but this file does not exist even on the host. (ubuntu 20.04)

it looks to me like you're expected to use `conntrack -L`  instead of reading this file.

see e.g. https://github.com/netdata/netdata/issues/161#issue-145492475

cc @aojea

Reply:
interesting, thanks for the pointer, it also  mentions

> I see, yes. And to be sure, I just confirmed that accessing /proc/net/stat/* does not put any load on the system, while accessing /proc/net/nf_conntrack directly takes a very very long time (40 seconds on a firewall with 100,000 tracked connection).

that I'd say is the cause of the flakiness in the TCP_CLOSE_WAIT test

I'll submit a new PR to change this

Reply:
https://github.com/kubernetes/kubernetes/pull/90687